dynamic.js

```diff
@@ -202,7 +204,13 @@ function consumesFormOnly (schema) {
   )
 }
 
-function getQueryParams (parameters, query) {
+function getQueryParams (parameters, query, fastifySchemas) {
+  if (typeof query === 'string' && query.match(/#$/)) {
```

Comment:
Would we have this problem also for the body if it is a string?

Reply:
Yes, it will be the problem with body, params, headers and all the stuff that accepts schemas as references.